iotedgedev/utility.py

```diff
@@ -93,7 +89,7 @@ def get_file_contents(file, expandvars=False):
                 return content
 
     def decode(self, val):
-        return val.decode("utf-8").strip()
```

Comment:
nit, though utf-8 may be default its nice to explicitly keep the encoding args because the next set of eyes  quickly see understand with less knowledge.